example/package.json

```diff
@@ -4,16 +4,17 @@
   "description": "Small example for messageformat-loader",
   "scripts": {
     "build": "webpack",
-    "postinstall": "cd .. && npm install && npm install --no-save messageformat",
+    "postinstall": "cd .. && npm install && npm install --no-save 'file:./example/node_modules/messageformat' 'file:./example/node_modules/webpack'",
```

Comment:
Uses symlinks so we get the version specified below instead of just latest necessarily.